tests/test_run_settings.py

```diff
@@ -208,18 +208,17 @@ def test_add_exe_args_list_of_mixed_lists_init():
     exe_args = [["1", "2", 3], ["4", "5", 6]]
 
     with pytest.raises(TypeError) as type_error:
-        settings = RunSettings("python", exe_args=exe_args)
+        _ = RunSettings("python", exe_args=exe_args)
 
-    assert "Executable arguments were not list of str or str" in type_error.value.args
+    assert "Executable arguments were not list of str or str." in type_error.value.args
 
 
 def test_add_exe_args_list_of_str_lists_init():
     """Ensure that list[list[str]] pass validation"""
     exe_args = [["1", "2", "3"], ["4", "5", "6"]]
 
-    settings = RunSettings("python", exe_args=exe_args)
-
-    assert settings.exe_args == exe_args
+    with pytest.raises(TypeError) as type_error:
```

Comment:
I'm not seeing a functional difference between this test and this previous.  What is it?